### PR TITLE
feat/seo

### PR DESCRIPTION
**Pull request by glazk0**

### CODE REVIEW

File: app/[lang]/blog/[slug]/page.tsx [hunk 1]
```diff
@@ -29,18 +29,15 @@ export async function generateMetadata({
...
-        },
-      ],
+      images: getURL(post.image!) || getURL('/assets/social.png'),
     },
     twitter: {
       card: 'summary_large_image',
       title: post.title!,
       description: replaceHTML(post.short!),
-      images: [getURL(post.image!) || getURL('/assets/social.png')],
+      images: getURL(post.image!) || getURL('/assets/social.png'),
```
Comment (lmachens) [inline app/[lang]/blog/[slug]/page.tsx:40]: I think `getURL` will always return a string. Thus, the fallback will never called.

**Review by lmachens — CHANGES_REQUESTED** [review]
looks good, only a small issue

File: app/[lang]/blog/[slug]/page.tsx (hunk 1 above)
Comment (glazk0) [reply app/[lang]/blog/[slug]/page.tsx:40]: So checking if it's blank would fix the issue?

Comment (lmachens) [reply app/[lang]/blog/[slug]/page.tsx:40]: `getURL(post.image || '/assets/social.png')`  should work

**Comment by glazk0** [comment]
There is a pnpm conflict but I let you fix it, don't wanna break stuff

**Review by lmachens — APPROVED** [review]
Good job!

**Pull request merged**